go/api/generic_commands.go

```diff
@@ -30,4 +30,69 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Unlink (delete) multiple keys from the database. A key is ignored if it does not exist.
+	// This command, similar to Del However, this command does not block the server
+	//
+	// Note:
+	//  When in cluster mode, the command may route to multiple nodes when keys map to different hash slots.
```

Comment:
There is new format of notice was introduced in https://github.com/valkey-io/valkey-glide/pull/2587. Please update it here. Reference: https://github.com/valkey-io/valkey-glide/pull/2738